README.md

```diff
@@ -62,99 +108,83 @@ exports.process = function process(msg, cfg, snapshot) { };
 ```
 
-## Fixtures
-Fixtures are used in testing code as a location to store test instances within a codebase.
+Furthermore, the provided fixture will be initially be checked against its respective `schema`, if the action/trigger has a static schema, for invalid message. This will not impede the running of the `process` action.
 
-For example, the ``process(msg, cfg)`` function takes at least 2 parameters:
+The CLI tool will print all details emitted from the `process`, and any values that have been returned before it exits.
 
-* msg: the message to be process by the component
-* cfg: component's configuration
+### Flags
 
-In order to execute your component, we need to know what parameters to pass to its ``process`` function. This is what the fixtures are for.
+`cmp:process` takes two optional flags:
+- -x, --fixture: fixture name to run against (optional)
+- -a, --action: name of action/trigger to run (optional)
 
-Fixtures are defined in a file ``test/fixture.json`` whereby the ``test`` folder is is expected to be located in the root of the component directory, i.e. in the same location as `component.json`.
+To view the help menu for this function, type `elasticio cmp:process -h`.
 
-A `fixtures.json` file has a root fixtures object, and then one or more defined fixtures. Below is an example file:
+### Examples
 
-````json
-{
-    "fixtures":{
-        "success":{
-            "msg":{
-                "headers":{},
-                "body":{}
-            },
-            "cfg":{
-                "calendarId":"{{GOOGLE_CALENDAR_ID}}",
-                "oauth":{
-                    "expires_in":3600,
-                    "token_type":"Bearer",
-                    "refresh_token":"{{GOOGLE_REFRESH_TOKEN}}",
-                    "access_token":"{{GOOGLE_ACCESS_TOKEN}}"
-                }
-            }
-        }
-    }
-}
-````
+`> elasticio cmp:process -x success -a lookupObject`
 
-The content of the file is a JSON object with a single key ``fixtures`` which contains named fixtures. Each fixture defines arguments to be passed to component's _process_ function: a message and configuration objects. 
+### Limitations
+None at the moment
 
-The only fixture in example above is named _success_. Please note that the  component's configuration may contain sensible data, such as API keys or OAuth tokens. Such data must not be placed inside fixture files because you will push them to your version control system, such as GitHub. Instead they should be replaced by variables using the [Handlebars](http://handlebarsjs.com/) syntax:
+## cmp:exec
 
-````
-"access_token":"{{google_access_token}}"
-````
+`cmp:exec` will allow you to run any exported function from an action/trigger that may be used on the platform, or `verifyCredentials` for a given component. For example, this allows you to run the functions `getMetaModel`, and other Select View functions. It can be run from within a component directory, or from outside by providing a path.
```

Comment:
```suggestion
`cmp:exec` will allow you to run `verifyCredentials` and any exported function from an action/trigger that may be used on the platform. For example, this allows you to run the functions `getMetaModel` and other Select View functions. It can be run from within a component directory, or from outside by providing a path.
```